src/misc/lv_style.h

```diff
@@ -43,6 +43,12 @@ extern "C" {
 #define LV_IMG_ZOOM_NONE            256        /*Value for not zooming the image*/
 LV_EXPORT_CONST_INT(LV_IMG_ZOOM_NONE);
 
+#if LV_USE_ASSERT_STYLE
+#define LV_STYLE_CONST_INIT(var_name, prop_array) const lv_style_t var_name = { .sentinel = LV_STYLE_SENTINEL_VALUE, .v_p = { .const_props = prop_array }, .has_group = 0xFF, .is_const = 1 }
```

Comment:
How about reuse sentinel to check the wrong usage(e.g. modify the const style)? we can add a new macro like LV_STYLE_SENTINEL_CONST.

Reply:
Ah; good idea. We may as well reuse those 4 bytes.

Reply:
Hmm, found a problem: the sentinel is only included if `LV_USE_ASSERT_STYLE` is enabled, otherwise it's removed from the structure entirely.

Reply:
Yes, I suppose the const check is only for debugging too.

Reply:
We ended up repurposing a bit from `prop1`. LVGL can only handle 512 properties anyway due to internal flags, so having 15 bits instead of 16 is not a problem.